vlib/strconv/format.md

```diff
@@ -0,0 +1,239 @@
+# v_printf/v_sprintf
+
+These are the implementation in v of the `printf` and `sprintf` functions used in C.
+
+***Note: These function are in C platform dependents, in V these are instead platform independents.***
+
+### v_sprintf
+
+`v_sprintf` has a variable number of parameters, the first is fixed and is a string that specify the format of the next parameters.
+
+```v
+import strconv
+fn main() {
+	a := "World"
+	s := strconv.v_sprintf("Hello %s!", a)
+	println(s)
+}
+```
+
+```
+Hello World!
+```
+
+### v_printf
+
+`v_printf` is the same of `v_sprintf` but print on the `stdout`, it doesn't return a string.
+
+### Syntax
+
+The syntax for a format placeholder is:
+
+```
+%[parameter][flags][width][.precision][length]type
+```
+
+#### Flags field
+
+The Flags field can be zero or more (in any order) of:
+
+| Character   | Description                                                  |
+| ----------- | ------------------------------------------------------------ |
+| `-` (minus) | Left-align the output of this placeholder. (The default is to right-align the output.) |
+| `+` (plus)  | Prepends a plus for positive signed-numeric types. positive = `+`, negative = `-`. (The default doesn't prepend anything in front of positive numbers.) |
+| `0` (zero)  | When the 'width' option is specified, prepends zeros for numeric types. (The default prepends spaces.) For example, `printf("%4X",3)` produces `   3`, while `printf("%04X",3)` produces `0003`. |
+
+#### Width field
+
+The Width field specifies a *maximum* number of characters to output, and is typically used to pad fixed-width fields in tabulated output, it cause truncation of oversized fields.
+
+The width field may be omitted, or a numeric integer value, or a dynamic value when passed as another argument when indicated by an asterisk `*`. For example, `v_printf("%*.s", 5, my_string)` will result in `   mystring` being printed, with a total width of 5 characters.
+
+#### Length field 
+
+The Length field can be omitted or be any of:
+
+| Character | Description                                                  |
+| --------- | ------------------------------------------------------------ |
+| `hh`      | For integer types, causes `printf` to expect an `byte`or `i8` argument. |
```

Comment:
Technically is should be "expect an" in all 4 lines in the table, as it is an article describing 'argument'.  "`byte` or `i8`" are extra words to narrow down the possibilities.

Also, make sure there is a space before "or" on all 4 lines.